src/attribute/attribute.controller.ts

```diff
@@ -1,31 +1,117 @@
-import { Controller, Get, Res, UseGuards } from '@nestjs/common';
+import { Body, ConflictException, Controller, Delete, Get, Param, Post, Put, Req, Res, UseGuards } from '@nestjs/common';
 import { AttributeService } from './attribute.service';
 import { ResponseService } from 'utils/response/customResponse';
 import { AuthGuard } from '@nestjs/passport';
+import { CreateAddressDto } from 'src/user/dto/create-address.dto';
+import { CreateAttributeDto } from './dto/attribute.dto';
+import { IdValidationPipe } from 'utils/validation/paramsValidation';
 
 @Controller('attribute')
 export class AttributeController {
   constructor(
     private readonly attributeService: AttributeService,
     private readonly responseService: ResponseService, // Inject the ResponseService
-  ) {}
+  ) { }
+
+  @Post()
+  @UseGuards(AuthGuard('jwt'))
+  async createAttribute(
+    @Body() createAttributeDto: CreateAttributeDto,
+    @Req() req,
+    @Res() res) {
+    try {
+     const data= await this.attributeService.create(createAttributeDto);
+      this.responseService.sendSuccess(res, "Attribute created Successfully",data);
```

Comment:
If we get data in return then only there should be a success response, here not such condition is present to verify data before sending a success response